app/scrapers/twitter.py

```diff
@@ -0,0 +1,24 @@
+import requests
+
+
+class Twitter():
+    """docstring for Twitter"""
+
+    def __init__(self):
+        self.loklakURL = 'http://api.loklak.org/api/search.json?\
+                            timezoneOffset=-330&q='
```

Comment:
PEP8 advises that we avoid backslashes like the plague that they are.  The problem is that one whitespace to the right of the backslash and the code stops working but the change is invisible to the reader.  The recommended approach is to use parenthesis: 

```python
self.loklakURL = ('http://api.loklak.org/api/search.json?'
                  'timezoneOffset=-330&q=')

Reply:
Also is timezoneOffset=-330 really essential?

Reply:
I don't think it's essential @cclauss, Can you please remove this @Remorax.

Reply:
Okay